csrc/device_lower/utils.cpp

```diff
@@ -45,53 +45,6 @@ kir::IfThenElse* cloneIfThenElse(kir::IfThenElse* ite) {
 
 namespace ir_utils {
 
-TVDomainGuard::TVDomainGuard(TensorView* tv, TensorDomain* td)
```

Comment:
Moved to ir/utils.h.

Many of the definitions in this file are actually not specific to device_lower. We should probably move to ir/utils.h.